### DISCUSSION TITLE
Add Bun :)

### DISCUSSION THREAD

**Discussion by DarrenBaldwin07** [remix-run/remix#7372] [Proposals]
What are the blockers for adding Bun alongside Deno in the `create-remix` CLI? Recently saw a PR merged from someone on their core team to support the bun package manager - would love to see more official support from the Remix side soon (especially since Bun 1.0 is releasing very soon)
![Screenshot 2023-09-07 at 10 41 39 PM](https://github.com/remix-run/remix/assets/68653294/e087a4e7-76ce-46f0-a30d-4cb4cf970593)

**Comment by sergiodxa** [comment]
In v2 that list of templates from the CLI will be removed, but Bun support would be interesting, I imagine someone needs to bulls an adapter and runtime package.

**Reply by hjonasson** [reply]
@sergiodxa @DarrenBaldwin07 FYI https://github.com/remix-run/remix/pull/7399
